src/store/modules/farmer/state.ts

```diff
@@ -0,0 +1,10 @@
+import { FarmerState, Farmer } from './types';
+import UTCDate from '../../../utils/UTCDate';
+
+const initialState: FarmerState = {
+  containsLocal: false,
+  farmers: [] as Farmer[],
+  lastModified: UTCDate.OLD_DATE,
+};
+
+export default initialState;
```

Comment:
New line at end of file please.

Reply:
:P There is a newline haha, Github shows lacking newlines with a little invalid sign at the end of the file, not by actually displaying the newline. I also have my editor setup to automatically format on save, so it will add the newline for me

Reply:
oh well shit, I commented that on every file :p